src/signin.js

```diff
@@ -998,6 +998,7 @@ function loginStatusBox (dom, listener, options) { // 20190630
   }
 
   var logoutButton = function (me, options) {
+    options = options || {}
```

Comment:
Not important, but you could've done this in the previous line as well, with `function (me, options = {})`